scripts/utils.js

```diff
@@ -0,0 +1,8 @@
+import fs from "fs-extra";
+
+export function getPackage() {
```

Comment:
Piggybacking off of a suggestion Greg made about my `sdk-logos` code- is it possible to import a package-lock file directly into `cdnCheck.js`?  Something like:

`import { dependencies } from '../package-lock.json`

If so we might be able to remove this function and simplify our code a bit.